src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -507,10 +513,19 @@ def load_acs_service_principals(config_path):
 
 def _create_kubernetes(resource_group_name, deployment_name, dns_name_prefix, name, ssh_key_value,
                        admin_username="azureuser", agent_count="3", agent_vm_size="Standard_D2_v2",
-                       location=None, service_principal=None, client_secret=None):
+                       location=None, service_principal=None, client_secret=None, windows=False, admin_password=''):
     from azure.mgmt.resource.resources.models import DeploymentProperties
     if not location:
         location = '[resourceGroup().location]'
+    windows_profile = None
+    os_type = 'Linux'
+    if windows:
+        windows_profile = {
+            "adminUsername": admin_username,
+            "adminPassword": admin_password,
```

Comment:
There's no validation in there yet (even for VMs) I'll try to add it in a different PR.